README.md

```diff
@@ -98,6 +98,10 @@ These operations take place in accordance with the rules explained in [this guid
 
 ## Changelog
 
+##### 4.1.13
+* Update Simple HTML library to the latest version 2.9.1
```

Comment:
The right name of the library should be "[PHP Simple HTML DOM Parser](https://sourceforge.net/projects/simplehtmldom/)" and the version we updated is 1.9.1 (not 2.9.1)